lib/ably/realtime/channel.rb

```diff
@@ -179,15 +184,26 @@ def presence
         @presence
       end
 
-      # Return the message history of the channel.
+      # Return the message history of the channel
+      #
+      # Once attached to a channel, you can retrieve messages published on the channel before the
```

Comment:
> Actually history is the only one not doing an implicit attach

I said this already; I think you're missing the point.

For this functionality to be relevant, you have to have called `subscribe()` before calling `history()`. There is no situation where calling `attach()` implicitly as part of the `history()` call makes any sense.